geometry_msgs/Pose goal
geometry_msgs/Pose[] path
---
bool sucess